cmdutil/debug/config.go

```diff
@@ -3,4 +3,29 @@ package debug
 // Config describes the configurable parameters for debugging.
 type Config struct {
 	Port int `env:"DEBUG_PORT,default=9999"`
+	PProfConfig
+}
+
+type PProfConfig struct {
```

Comment:
The `PProfConfig` is stutter when it's embedded like this. Why not call the embedded config struct `PProf`.